nextcord/flags.py

```diff
@@ -486,6 +487,58 @@ def all(self) -> List[UserFlags]:
         return [public_flag for public_flag in UserFlags if self._has_flag(public_flag.value)]
 
 
+@fill_with_flags()
+class MemberFlags(BaseFlags):
+    r"""Wraps up the Discord Guild Member flags
+
+    .. versionadded:: 2.6
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two MemberFlags are equal.
+
+        .. describe:: x != y
+
+            Checks if two MemberFlags are not equal.
+        .. describe:: hash(x)
+
+            Return the flag's hash.
```

Comment:
```suggestion
            Returns the flag's hash.
```
Same form as the other operations: "Returns", "Checks".